config/sidekiq_scheduler.yml

```diff
@@ -273,3 +273,8 @@ TestUserDashboard::MirrorAccountsInBigQuery:
   cron: "0 * * * * America/New_York"
   class: TestUserDashboard::MirrorAccountsInBigQuery
   description: "Mirrors TUD accounts in BigQuery."
+
+TestUserDashboard::MirrorAccountAvailabilityLogsInBigQuery:
+  cron: "0 */2 * * * America/New_York"
+  class: TestUserDashboard::MirrorAccountAvailabilityLogsInBigQuery
+  description: "Mirrors the test_user_dashboard_tud_account_availability_logs table in BigQuery." 
```

Comment:
Could we get a new line char here please?

Reply:
Yes :-)